examples/rails-4.2.5/Gemfile

```diff
@@ -1,11 +1,12 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.5'
+gem 'rails', '~> 4.2.5'
 
 gem 'rails-api'
 
 gem 'spring', :group => :development
-
+gem 'rake', '< 11.0'
+gem 'sprockets-rails', ' ~> 2.3.3'
```

Comment:
What was happening before we added these in?

Reply:
@aalong-tr I am not sure - it was before my time.

The task threw errors, so I upgraded Rails to 4.2.11, and that required other additions to the Gemfile.